README.md

```diff
@@ -94,6 +94,8 @@ docker tag app1 <your-docker-hub-id>/app1:v1
 docker push <your-docker-hub-id>/app1:v1
 ```
 
+You can create your own on hub.docker.com with a new account on a free plan and would need to run `docker login` successfully before running the previous commands. A new repository on your Docker Hub account will be created after success.
```

Comment:
The final step is to publish our image to a remote Docker registry. The app used for this Demo is already available as the image `jeanfrg/app1`, however to publish your own you'll need to create an account on [hub.docker.com ](https://hub.docker.com). Then you'll need to run `docker login`.

Once this has been done we need to use these commands to publish a new image. A new repository on your Docker Hub account will be created.

```bash
docker tag app1 <your-docker-hub-id>/app1:v1
docker push <your-docker-hub-id>/app1:v1
```

Reply:
sent new PR for this